plugins/dbnd-docker/src/dbnd_docker/kubernetes/kubernetes_engine_config.py

```diff
@@ -92,15 +116,18 @@ class KubernetesEngineConfig(ContainerEngineConfig):
     requests = parameter.none()[Dict]
     limits = parameter.none()[Dict]
 
-    pod_exit_code_to_retry_count = parameter(empty_default=True).help(
-        "Mapping between pod exit code to amount of pod retry attempts"
-    )[Dict]
-    pod_retry_delay = parameter.help(
-        "The delay between each pod retry attempt in time delta format. 1m, 5s, 1h, etc."
+    pod_error_cfg = parameter(
+        default=PodRetryConfiguration(
```

Comment:
Can you move the default values to `databand-core.cfg` instead? They are more accessible to the user there, and will give an example, making it easier to add and change it

Reply:
I can but is it possible to have a python object in `databand-core.cfg`?